test/unit-test/test-Model.js

```diff
@@ -6836,6 +6836,596 @@ describe('Unit Test/Model Test', function() {
         });
       });
     });
+
+    it('"4-D tensor as input0 that is TENSOR_FLOAT32 type and 1-D tensor as input1 that is TENSOR_INT32 type" are ok for "BATCH_TO_SPACE_ND" operation', function() {
```

Comment:
@cuiyanx According to https://github.com/intel/webml-polyfill/blob/master/src/nn/Enums.js#L1199, **All values of 1-D Tensor must be >= 1.** , we should add this info into test case purpose.